### DISCUSSION TITLE
Data of square following Circular Shape

### DISCUSSION THREAD

**Discussion by kushaldgowda** [DLR-SL/CPACS#749] [Q&A] [answered]
I used just 5 points data for square fuselage format.
But It is showing like circular only.
![Screenshot (23)](https://user-images.githubusercontent.com/93525414/140603887-f74214f9-d7fd-4058-9569-7ca3d56d233a.png)

How to get a shape of square for fuselage?
[Square.zip](https://github.com/DLR-SL/CPACS/files/7490131/Square.zip)

**Comment by MarAlder** [comment, marked as answer]
Using the [`pointListElement`](https://www.cpacs.de/documentation/CPACS_3_3_0_Docs/html/d0271c9a-2b38-36af-5f3f-50d49dabdbda.htm) you need to provide much more points to approximate a square profile. Since CPACS 3.3 you can also use kinks. This is, however, not yet implemented in CpacsCreator.

An easier way to define square fuselages is to use the [`standardProfile`](https://www.cpacs.de/documentation/CPACS_3_3_0_Docs/html/3d9a2aa3-f4c6-be22-2fc6-359f3ecc25e6.htm) element instead of [`pointListElement`](https://www.cpacs.de/documentation/CPACS_3_3_0_Docs/html/d0271c9a-2b38-36af-5f3f-50d49dabdbda.htm). But this still needs to be implemented in TiGL. 

So, you should go for a list of many points to approximate the square.
